lib/ansible/modules/network/cpm/cpm_status.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# (C) 2018 Red Hat Inc.
+# Copyright (C) 2018 Western Telematic Inc.
+#
+# GNU General Public License v3.0+
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+#
+# Module to execute CPM Status Commands on WTI OOB and PDU devices.
+# CPM Networking
+#
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = """
+---
+module: cpm_status
+version_added: "2.7"
+author: "Western Telematic Inc. (@wtinetworkgear)"
+short_description: Get status and parameters from WTI OOB and PDU devices
+description:
+    - "Get various status and parameters from WTI OOB and PDU devices"
+options:
+  cpm_action:
+    description:
+      - This is the Action to send the module.
+    required: true
+    choices: [ "temperature", "firmware", "status", "alarms" ]
+  cpm_url:
+    description:
+      - This is the URL of the WTI device to send the module.
+    required: true
+  cpm_username:
+    description:
+      - This is the Username of the WTI device to send the module.
+  cpm_password:
+    description:
+      - This is the Password of the WTI device to send the module.
+  use_https:
+    description:
+      - Designates to use an https connection or http connection.
+    required: false
+    type: bool
+    default: True
```

Comment:
Is it possible that some WTI devices may have self-signed certs, if so please add
```yaml
  validate_certs:
    description:
      - If C(no), SSL certificates will not be validated. This should only be used
        on personally controlled sites using self-signed certificates.
    default: yes
    type: bool
```
and corresponding code